plugin/al/plugin/AL_USDMayaTestPlugin/AL/usdmaya/commands/test_ProxyShapeSelect.cpp

```diff
@@ -1359,188 +1643,218 @@ TEST(ProxyShapeSelect, pseudoRootSelect)
-  EXPECT_EQ(1u, proxy->selectedPaths().count(SdfPath("/root/hip1/knee1/ankle1/ltoe1")));
-  EXPECT_EQ(1u, proxy->selectedPaths().count(SdfPath("/root/hip2")));
-  EXPECT_TRUE(proxy->isRequiredPath(SdfPath("/root")));
-  EXPECT_TRUE(proxy->isRequiredPath(SdfPath("/root/hip1")));
-  EXPECT_TRUE(proxy->isRequiredPath(SdfPath("/root/hip1/knee1")));
-  EXPECT_TRUE(proxy->isRequiredPath(SdfPath("/root/hip1/knee1/ankle1")));
-  EXPECT_TRUE(proxy->isRequiredPath(SdfPath("/root/hip1/knee1/ankle1/ltoe1")));
-  EXPECT_TRUE(proxy->isRequiredPath(SdfPath("/root/hip2")));
-  // Ensure the proxyShape is no longer selected
-  { SCOPED_TRACE(""); compareNodes({SdfPath("/root/hip1/knee1/ankle1/ltoe1"), SdfPath("/root/hip2")}); };
-
-  // Undo the remove
-  MGlobal::executeCommand("undo", false, true);
-
-  // State 3: ltoe1, proxy, hip2 selected
-  // make sure ltoe1 and hip2 are contained in the selected paths (for hydra selection)
-  EXPECT_EQ(2u, proxy->selectedPaths().size());
-  EXPECT_EQ(1u, proxy->selectedPaths().count(SdfPath("/root/hip1/knee1/ankle1/ltoe1")));
-  EXPECT_EQ(1u, proxy->selectedPaths().count(SdfPath("/root/hip2")));
-  EXPECT_TRUE(proxy->isRequiredPath(SdfPath("/root")));
-  EXPECT_TRUE(proxy->isRequiredPath(SdfPath("/root/hip1")));
-  EXPECT_TRUE(proxy->isRequiredPath(SdfPath("/root/hip1/knee1")));
-  EXPECT_TRUE(proxy->isRequiredPath(SdfPath("/root/hip1/knee1/ankle1")));
-  EXPECT_TRUE(proxy->isRequiredPath(SdfPath("/root/hip1/knee1/ankle1/ltoe1")));
-  EXPECT_TRUE(proxy->isRequiredPath(SdfPath("/root/hip2")));
-  // Ensure the proxyShape is selected again, as well as ltoe2 and hip2
-  { SCOPED_TRACE(""); compareNodes({SdfPath("/root/hip1/knee1/ankle1/ltoe1"), SdfPath("/"), SdfPath("/root/hip2")}); };
-
-  // Undo the append
-  MGlobal::executeCommand("undo", false, true);
-
-  // State 2: ltoe1 selected
-  // make sure ltoe1 is contained in the selected paths (for hydra selection)
-  EXPECT_EQ(1u, proxy->selectedPaths().size());
-  EXPECT_EQ(1u, proxy->selectedPaths().count(SdfPath("/root/hip1/knee1/ankle1/ltoe1")));
-  EXPECT_TRUE(proxy->isRequiredPath(SdfPath("/root")));
-  EXPECT_TRUE(proxy->isRequiredPath(SdfPath("/root/hip1")));
-  EXPECT_TRUE(proxy->isRequiredPath(SdfPath("/root/hip1/knee1")));
-  EXPECT_TRUE(proxy->isRequiredPath(SdfPath("/root/hip1/knee1/ankle1")));
-  EXPECT_TRUE(proxy->isRequiredPath(SdfPath("/root/hip1/knee1/ankle1/ltoe1")));
-  // Ensure the proxyShape is no longer selected, and the other sub-path is
-  { SCOPED_TRACE(""); compareNodes({SdfPath("/root/hip1/knee1/ankle1/ltoe1")}); };
-
-  // Undo the toggle
-  MGlobal::executeCommand("undo", false, true);
-
-  // State 1: proxy selected
-  EXPECT_EQ(0u, proxy->selectedPaths().size());
-  { SCOPED_TRACE(""); compareNodes({SdfPath("/")}); };
-
-  // Undo the initial replace
-  MGlobal::executeCommand("undo", false, true);
+  //
+  // RB: Disabling this test for now as this is currently broken in UFE pending some feedback. 
```

Comment:
If all else fails, you can always select Maya nodes using the Maya API, or the select command, which will automatically update the UFE selection.